src/manifests_workflow/input_manifests.py

```diff
@@ -164,7 +164,12 @@ def add_to_cron(self, version: str) -> None:
         with open(jenkinsfile, "r") as f:
             data = f.read()
 
-        cron_entry = f"H 1 * * * %INPUT_MANIFEST={version}/{self.prefix}-{version}.yml;TARGET_JOB_NAME=distribution-build-{self.prefix}\n"
+        build_platform_dict = {"opensearch": "linux macos windows", "opensearch-dashboards": "linux windows"}
+        build_platform = build_platform_dict.get(self.prefix, "linux macos windows")
```

Comment:
NIT: Hey @zelinh, thanks for the PR,  can we get set the default to just `linux` ?

Reply:
+1 I believe this will fail for 1.x version where windows is not supported yet

Reply:
@prudhvigodithi hi, as the discussion in this issue https://github.com/opensearch-project/opensearch-build/issues/2912, I think having three as default would be the target in long run. We are also planning to support windows for 1.x series but haven't decided which version to start. No strong objective though since this default won't be used in most cases. I would update this to `linux`.